library/Notifications/Widget/ItemList/EventRuleList.php

```diff
@@ -17,4 +18,10 @@ protected function getItemClass(): string
     {
         return EventRuleListItem::class;
     }
+
+    protected function init(): void
+    {
+        $this->getAttributes()
+            ->set('data-icinga-detail-url', (string) Url::fromPath('notifications/event-rule'));
```

Comment:
Same change as for `ContactGroupList`.
Remember to introduce null values to `Links::eventRule()` as well.